egs/wsj/s5/steps/libs/nnet3/xconfig/basic_layers.py

```diff
@@ -999,7 +999,7 @@ def set_default_configs(self):
                        'dim': -1,
                        'cepstral-lifter': 22.0,
                        'affine-transform-file': '',
-                       'write-init-config': True}
+                       'write-init-config': False}
```

Comment:
Its a bug, I guess. It can be removed entirely. The option and also the condition check in get_full_config.